src/amo/components/CategoryHead/index.js

```diff
@@ -0,0 +1,83 @@
+/* @flow */
+import config from 'config';
+import invariant from 'invariant';
+import * as React from 'react';
+import Helmet from 'react-helmet';
+import { connect } from 'react-redux';
+import { compose } from 'redux';
+
+import { getCanonicalURL } from 'amo/utils';
+import { ADDON_TYPE_EXTENSION, ADDON_TYPE_THEME } from 'core/constants';
+import translate from 'core/i18n/translate';
+import type { AppState } from 'amo/store';
+import type { CategoryType } from 'amo/types/categories';
+import type { I18nType } from 'core/types/i18n';
+
+type Props = {|
+  category: CategoryType | null,
+  type: 'extension' | 'theme',
```

Comment:
shouldn't type be extension or persona?

Reply:
Indeed, but I guess we could even remove this prop. Each category has a `type`.

Reply:
yep that's better 👍